tests/spice_test.go

```diff
@@ -63,7 +63,7 @@ var _ = Describe("Vmlifecycle", func() {
 			Expect(section.Key("port").Value()).To(Equal("4000"))
 			Expect(section.HasKey("proxy")).To(BeTrue())
 			close(done)
-		}, 30)
+		}, 50)
```

Comment:
I am not so sure where to use constants in the tests. Maybe one solution would be to define a standard test run time with a constant, and then multiply it, first with a test run factor  and then with a system speed factor?